autorest/codegen/templates/request_builders.py.jinja2

```diff
@@ -7,7 +7,7 @@
 {% if not is_python3_file %}
 # fmt: off
 {% endif %}
-{% for request_builder in request_builders %}
+{% for request_builder in request_builders | selectattr("abstract", "ne", True) %}
```

Comment:
for these operations / request builders, I think it would be cleaner to choose which ones to pass in the serializers. General rule of thumb I follow is to have less code in the templates, bc jinja is not as easy to read / maintain